apps/i18n/common/en_us.json

```diff
@@ -1488,6 +1488,10 @@
   "incubator_codebridge_python_desc": "Try out Code.org's first Python Code Editor.",
   "incubator_codebridge_python_project_button": "Create Your Own Project",
   "incubator_codebridge_python_title": "Codebridge: Python",
+  "incubator_genai_title": "Exploring Generative AI",
+  "incubator_genai_desc": "Try our new course that prepares students to explore and create with generative AI, developing critical thinking and ethical awareness",
```

Comment:
This is a super nit but it looks like the other descriptions have periods at the end; can you add one here?

Reply:
Makes total sense, I've added that period to bring it more in line with the others